README_zh.md

```diff
@@ -0,0 +1,727 @@
+_Only the original [README] is guaranteed to be up-to-date._
+
+只有原版的[README]是实时更新的。
```

Comment:
Can you propose a _suggestion_ ( press the first button on the toolbar or you can use <kbd>ctrl</kbd>+<kbd>g</kbd> ) so that we can use the review function to change the file?

Reply:
Done. Converted most comments to _suggestion_. Probably will do the "mirroring" part later today.